dspace-spring-rest/src/main/java/org/dspace/app/rest/exception/DSpaceApiExceptionControllerAdvice.java

```diff
@@ -50,6 +50,17 @@ protected void handleIOException(HttpServletRequest request, HttpServletResponse
                           HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
     }
 
+    @ExceptionHandler(IllegalArgumentException.class)
```

Comment:
this could be a bit too much extensive... an illegal argument ex in an internal method could be completely unrelated to the parameter supplied by the rest client